std_msgs/Time time
geometry_msgs/Point32[] pixelLocations